IDE/Renesas/e2studio/RZN2L/flash_app/src/Flash_wrappedkey_public.s

```diff
@@ -0,0 +1,30 @@
+/* Flash_sections.s
+ *
+ * Custom configuration for wolfCrypt/wolfSSL.
```

Comment:
Does this comment apply?

Reply:
No, it doesn't. Will remove the line. Thanks.